tier1-icons.html

```diff
@@ -354,10 +354,10 @@
         <g id="subscribe-hollow">
           <path d="M9 3.966l1.167 2.49.453.964 1.054.162 2.75.423-2.056 2.112-.7.722.162.994.47 2.88-2.333-1.288L9 12.89l-.97.536-2.33 1.287.47-2.877.163-1-.7-.723-2.057-2.107 2.747-.423 1.054-.163.453-.965L9 3.965M9 0a.552.552 0 0 0-.48.277l-2.5 5.33-5.572.857a.517.517 0 0 0-.418.36.5.5 0 0 0 .12.528l4.05 4.16-.96 5.87a.53.53 0 0 0 .776.548L9 15.176l4.98 2.754a.472.472 0 0 0 .246.07h.01a.5.5 0 0 0 .3-.1.517.517 0 0 0 .22-.52l-.957-5.867 4.048-4.16a.5.5 0 0 0 .12-.53.517.517 0 0 0-.42-.36l-5.572-.857-2.5-5.33A.552.552 0 0 0 9 0z"/>
         </g>
-        <g id="surveys">
+        <g id="surveys" mirror-rtl="true">
           <path d="M15 2h-2a2.006 2.006 0 0 0-2-2H7a2.006 2.006 0 0 0-2 2H3a2.006 2.006 0 0 0-2 2v12a2.006 2.006 0 0 0 2 2h12a2.006 2.006 0 0 0 2-2V4a2.006 2.006 0 0 0-2-2zM4 15a1 1 0 1 1 1-1 1 1 0 0 1-1 1zm0-4a1 1 0 1 1 1-1 1 1 0 0 1-1 1zm0-4a1 1 0 1 1 1-1 1 1 0 0 1-1 1zm10 8H8a1 1 0 0 1 0-2h6a1 1 0 0 1 0 2zm0-4H8a1 1 0 0 1 0-2h6a1 1 0 0 1 0 2zm0-4H8a1 1 0 0 1 0-2h6a1 1 0 0 1 0 2z"/>
         </g>
-        <g id="table-of-contents">
+        <g id="table-of-contents" mirror-rtl="true">
           <path d="M17 0H1a1 1 0 0 0-1 1v16a1 1 0 0 0 1 1h16a1 1 0 0 0 1-1V1a1 1 0 0 0-1-1zM3 16a1 1 0 0 1 0-2 1 1 0 0 1 0 2zm0-4a.945.945 0 0 1-1-1 .945.945 0 0 1 1-1 .945.945 0 0 1 1 1 .945.945 0 0 1-1 1zm0-4a.945.945 0 0 1-1-1 .945.945 0 0 1 1-1 .945.945 0 0 1 1 1 .945.945 0 0 1-1 1zm0-4a.945.945 0 0 1-1-1 .945.945 0 0 1 1-1 .945.945 0 0 1 1 1 .945.945 0 0 1-1 1zm12 12H7a1 1 0 0 1 0-2h8a1 1 0 0 1 0 2zm0-4H7a1 1 0 0 1 0-2h8a1 1 0 0 1 0 2zm0-4H7a1 1 0 0 1 0-2h8a1 1 0 0 1 0 2zm0-4H7a1 1 0 0 1 0-2h8a1 1 0 0 1 0 2z"/>
         </g>
         <g id="tag">
```

Comment:
I think it could go either way... if the orientation is tied to the UI in some way then I agree it should be mirrored, but if it's just kind of random and is more representative of how you might hold the tag in your hand, then no.